task/buildah/0.3/buildah.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     description: Reference of the image buildah will produce.
   - name: BUILDER_IMAGE
     description: The location of the buildah builder image.
-    default: quay.io/buildah/stable:v1.18.0
```

Comment:
the README may need to be updated too.

Reply:
done.

Reply:
i think someone else updated it too this morning, so you may need rebase...